pkg/autoscaler/fake/fake_metric_client.go

```diff
@@ -17,9 +17,28 @@ limitations under the License.
 package fake
 
 import (
+	"fmt"
 	"time"
 
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
+	kubeinformers "k8s.io/client-go/informers"
+	fakeK8s "k8s.io/client-go/kubernetes/fake"
```

Comment:
We are wildly inconsistent with this import across the codebase, but the suggested pattern seems to be both common and fits with the lower-case imports

```suggestion
	fakek8s "k8s.io/client-go/kubernetes/fake"
```